fix(ic-dashboard): Ledger API cursor names differ; verify step uses v4 (#328)
dfinity/icskills: skills/ic-dashboard/SKILL.md

1. **Prefer v2+ APIs with cursor pagination.** IC API v4 (`/api/v4/canisters`, `/api/v4/subnets`), ICRC API v2 (`/api/v2/ledgers`, `/api/v2/ledgers/{id}/transactions`, etc.), and SNS API v2 (`/api/v2/snses`, `/api/v2/snses/{id}/proposals`, `/api/v2/snses/{id}/neurons`) use `after`, `before`, and `limit` for stable, efficient paging. Avoid v1/offset-based endpoints when a v2+ alternative exists.
2. **Choose the right API** for the data you need: IC API (canisters, subnets, NNS neurons/proposals), **Ledger API for mainnet ICP** (accounts, transactions, supply), ICRC API for **other** ICRC ledgers only (ckBTC, SNS tokens, testicp — ICRC API does not expose mainnet ICP), SNS API (SNS list, neurons, proposals), Metrics API (governance, cycles, Bitcoin, etc.).
3. **Use the OpenAPI spec** to get exact path, query, and body schemas and response shapes; prefer the spec over hand-written docs to avoid drift.

> 4. **Call over HTTPS** with `GET` (or documented method). Use the cursor fields from v2+ responses to request the next or previous page. The names are not uniform: IC, ICRC and SNS APIs return `next_cursor` / `previous_cursor`, but **Ledger API returns `next_cursor` / `prev_cursor`** and puts the rows under `accounts` rather than `data`. Read the field names from the response (or the OpenAPI spec) rather than assuming.

Reviewer: `/v2/accounts` returns rows under `accounts` but `/v2/transactions` returns them under `blocks`, so the key varies by endpoint rather than being a Ledger-wide accounts.

```suggestion
4. **Call over HTTPS** with `GET` (or documented method). Use the cursor fields from v2+ responses to request the next or previous page. The names are not uniform: IC, ICRC and SNS APIs return `next_cursor` / `previous_cursor`, but **Ledger API returns `next_cursor` / `prev_cursor`** and names the row array after the resource (`accounts`, `blocks`, …) rather than a uniform `data`. Read the field names from the response (or the OpenAPI spec) rather than assuming.
```
